cli_tools/common/mount/inspector.go

```diff
@@ -66,7 +66,7 @@ func (mi *defaultMountInspector) Inspect(dir string) (mountInfo InspectionResult
 
 // getDeviceForMount returns the path of the block device that is mounted for dir.
 func (mi *defaultMountInspector) getDeviceForMount(dir string) (string, error) {
-	stdout, err := mi.shellExecutor.Exec("getDeviceForMount", "--noheadings", "--output=SOURCE", dir)
+	stdout, err := mi.shellExecutor.Exec("findmnt", "--noheadings", "--output=SOURCE", dir)
```

Comment:
the only thing i have to say here is that you don't need tools like findmnt when you're in go. there are packages to do this because the list of mounted filesystems is available to every process at /proc/self/mounts. unsure if the change is appropriate to this PR but just as a suggestion, minimizing execs usually produces more reliable code :)

Reply:
I think this is a good suggestion; lacking tests that @zoran15 mentioned, I wouldn't be confident that my custom-parsing logic for `/proc/self/mounts` would work everywhere. (Although I would hope that the format of this file would be consistent across distros and versions.)

Reply:
i meant to use an existing 3p golang module for it, but yes the format is stable and provided by the kernel directly.